client-tags/typing.md

```diff
@@ -39,9 +39,9 @@ A typing notification is represented by a [`TAGMSG`][tags] command sent with a `
 
 Input event handlers MUST be throttled so that any `typing` notification is not sent within 3 seconds of another one for a given target.
 
-After receiving a typing notification, clients SHOULD assume the sender is still typing until any of the following conditions is met:
+After receiving a typing notification, clients SHOULD assume the sender is still typing until any of the following conditions is met on the same target (channel or private message):
 
-* A message is received from the sender
+* A `PRIVMSG` or `NOTICE` is received from the sender
```

Comment:
Don't think this needs to be changed. "message" is already defined further up as:
> For the purpose of this specification, a message is defined as a PRIVMSG, NOTICE, TAGMSG, or batch end.

Reply:
Ok. Feels weird to include TAGMSG though...